hoot-core/src/main/cpp/hoot/core/conflate/MarkForReviewMerger.cpp

```diff
@@ -62,7 +60,7 @@ void MarkForReviewMerger::apply(const OsmMapPtr& map,
 
   if (_eids.size() >= 1)
   {
-    ReviewMarker().mark(map, _eids, _note, _reviewType, _score);
+    _reviewMarker.mark(map, _eids, _note, _reviewType, _score);
```

Comment:
I removed the static specifier from ReviewMarker::mark in order to more easily instantiate with a custom config opt.  We actually were calling that method in a non-static fashion all over the place anyway.  Additionally, I went through and made this type of change to reduce the number of unnecessary ReviewMarker instantiations.

Reply:
I like this change!